index.js

```diff
@@ -865,10 +855,8 @@ class Batch {
   }
 }
 
-class Watcher extends EventEmitter {
+class Watcher {
```

Comment:
Now that it's no longer inheriting from EventEmitter, could consider inheriting from ReadyResource to simplify the ready-close logic